pkg/policy/id_test.go

```diff
@@ -29,13 +29,15 @@ func TestIDIsValid(t *testing.T) {
 	}{
 		{ID("DenyEncryptionSt1"), true},
 		{ID(""), true},
+		{ID("abc-test"), true},
+		{ID("abc test"), true},
```

Comment:
These tests are not useful according to https://docs.aws.amazon.com/AmazonS3/latest/dev/example-bucket-policies.html.  Only three cases like empty string, non-empty string and non-utf8 string are good enough.